napari/_qt/_qapp_model/qactions/_layerlist_context.py

```diff
@@ -104,7 +120,9 @@ def _paste_spatial_from_clipboard(ll: LayerList) -> None:
     for layer in ll.selection:
         for key in loaded:
             loaded_attr_value = loaded[key]
-            if isinstance(loaded_attr_value, list):
+            if key == 'units':
+                loaded_attr_value = loaded_attr_value[-layer.ndim :]
```

Comment:
why is it ok here to only get the last ndim values for units? Are we assuming that if the layer we copied from has more dimensions than the layer we're pasting to, the shared dimensions are the last ones? Just for my own understanding.

Reply:
Those are good questions, I didn't think to try mismatched dimensions. I'm also interested in the answer

Reply:
In general, if the target layer has fewer dimensions than the source one, we need to decide which use. 
As we do not have named axis yet, I follow here the viewer semantic and get information from last axes. 

Same as in line 137 of this file 

(or on main)
https://github.com/napari/napari/blob/8a958c94ffb5b8a3cb6d72f41b490823fdac6636/napari/_qt/_qapp_model/qactions/_layerlist_context.py#L119